docs/adrs/00004-importer-scalability.md

```diff
@@ -0,0 +1,62 @@
+# 00004. Importer Scalability in Trustify
+
+Date: 2025-02-14
+
+## Status
+
+Draft
+
+## Context
+
+This ADR is concerned with our ability to scale the importer processes
+that ingest documents -- SBOM's and advisories -- from various
+sources.
+
+Currently, the importer process iterates over the enabled importer
+definitions in the database sequentially, running each according to
```

Comment:
we should identify what risks there are changing from predictable sequential processing to parallel processing eg. increase in RAM/CPU, db conn ... this change should be well tested against data at scale so we understand impact.